RadarSDK/Radar.m

```diff
@@ -43,6 +43,12 @@ + (id)sharedInstance {
 
 + (void)initializeWithPublishableKey:(NSString *)publishableKey {
     [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo type:RadarLogTypeSDKCall message:@"initialize()"];
+    
+    Class RadarSDKMotion = NSClassFromString(@"RadarSDKMotion");
+    if (RadarSDKMotion) {
+        id radarSDKMotion = [[RadarSDKMotion alloc] init];
+        [RadarActivityManager sharedInstance].radarSDKMotion = radarSDKMotion;
+    }
```

Comment:
Thinking out loud about someone deciding to turn this on: basically, in the first build that they import `RadarSDKMotion`, they'll have to add the new permission.

Given that we have this check above, we'll not need to do any version gating on `useLocationMetadata`, right?

Reply:
For this chunk of code, we are not concerned about if the flags are set to allow this feature, we are just loading a side component if we are able too. We also do not even have the feature flags at this point.

Right now, its the responsibility of ActivityManager and LocationManager to decide if we want to perform these extra data collection based on the feature flags.